docs/src/main/asciidoc/keycloak-guide.adoc

```diff
@@ -371,48 +304,6 @@ export access_token=$(\
 )
 ```
 
-== Multi-Tenant Deployments
```

Comment:
We don't support that anymore?

Reply:
Nope. That was specific to the Keycloak Undertow adaptor.